.travis.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+language: node_js
+os:
+  - linux
+  - osx
+
+node_js:
+  - "10"
+  - "10.15"
+  - "10.19"
+  - "10.20"
```

Comment:
`10` above will run the latest `10.x` version, which is `10.20.1` looking at the [Travis job](https://travis-ci.com/github/adobe/node-httptransfer/jobs/323161705#L189), so we can drop `10.20` here.